versions.mk

```diff
@@ -16,4 +16,4 @@ MDBOOK_VERSION := v0.4.5
 PLANTUML_VERSION := 1.2020.16
 GH_VERSION := 2.7.0
 CERT_MANAGER_VERSION := v1.10.1
-CAPI_VERSION := v1.3.4
```

Comment:
Curious what is the usage of versions.mk file? Just for users to check versions information?

Reply:
Yes correct, some of them remains outdated but I think we should be more careful that this file serves its purpose.